scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -484,6 +485,7 @@ task CollectSampleQualityMetrics {
     >>>
 
     runtime {
+        docker: "${gatk_docker}"
```

Comment:
Change to:
```
docker: gatk_docker
```
This will make the upgrade to WDL 1.0 just a bit easier.

Reply:
Replaced here and in all occurrences of gCNV WDLs